ansible/fosdem_hosts

```diff
@@ -1,4 +1,4 @@
 [website]
 
 [event-primary]
-151.216.191.253
+conference.fosdem.net
```

Comment:
The DNS zone is hosted by this server, so if bind breaks recovery will be impeded. I'd suggest using `ansible_host=151.216.191.253` here so we always connect by IP